apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -282,6 +329,26 @@ export default function CourseOfferingEditor(props) {
         showPreview={true}
         helpTipText={'Image used to market the curriculum around the site.'}
       />
+      <label style={styles.videoContainer}>
+        Video
+        <HelpTip>
+          <p>
+            Search and select the corresponding video for the course offering.
+          </p>
+        </HelpTip>
+        <div style={{width: '75%'}}>
+          <Select
+            options={renderedOptions}
+            filterOption={filterOption}
+            value={
+              courseOffering.video === null ? 'None' : courseOffering.video
+            }
+            onChange={handleVideoSelection}
+            defaultValue={translatedNoneOption}
+          />
+        </div>
+        {thumbnail && <img src={thumbnail} style={styles.image} />}
```

Comment:
Nit: can you add an `alt` text field to the `img` component please (could just be blank as "")